### DISCUSSION TITLE
Escape markdoc tags

### DISCUSSION THREAD

**Discussion by ntgussoni** [markdoc/markdoc#141] [Q&A] [answered]
Is there any way to escape tags?

for example, while building documentation I want to do this:
~~~
```
{% foo %}
```
~~~

I want this to be rendered as is, but it's trying to parse the foo node.

**Comment by matv-stripe** [comment, marked as answer]
@ntgussoni Hi!

If you add `process=false` it should prevent them from parsing them as nodes: 

````
``` {% process=false %}
{% foo %}
```
````

**Reply by ntgussoni** [reply]
Brilliant, that worked. is this supposed to work on any macro?
